src/main/java/org/breedinginsight/brapps/importer/services/MappingManager.java

```diff
@@ -53,6 +53,12 @@ public class MappingManager {
 
     private ImportConfigManager configManager;
 
+    public static String wrongDataTypeMsg = "Column name \"%s\" must be integer type, but non-integer type provided.";
+    public static String blankRequiredField = "Required field \"%s\" cannot contain empty values";
+    public static String missingColumn = "Column name \"%s\" does not exist in file";
+    public static String missingUserInput = "User input, \"%s\" is required";
+    public static String wrongUserInputDataType = "User input, \"%s\" must be an %s";
```

Comment:
Unused, remove. It looks like the code uses the methods defined in `BrAPIImportService` to generate error messages instead.

Reply:
Yep, it was my intention to base this change off of develop, not 1830. I'm closing this PR and issuing a new one: [BI-1960-2](https://github.com/Breeding-Insight/bi-api/pull/316).